Formula/mtoc.rb

```diff
@@ -20,6 +20,7 @@ class Mtoc < Formula
 
   depends_on "llvm" => :build
   depends_on :macos
+  conflicts_with "ocmtoc", because: "both install `mtoc` binaries"
```

Comment:
Please add `conflicts_with` in a separate PR after this one is merged so we don't have to rebuild the `mtoc` bottles.

Reply:
Done.